javascript-data-model/order-history.js

```diff
@@ -0,0 +1,102 @@
+var item = { };
+var person = 'JS Masher';
+var orderHistory = [
+  item.orderOne = {
+    orders: 1,
+    name: 'JavaScript for impatient programmers',
+    image: '',
+    type: 'Book',
+    isbn: null,
+    author: 'Rauschmayer, Dr. Axel',
+    price: 31.55,
+    total: 34.00,
+    ship: person,
+    orderPlaced: 'August 4, 2020',
+    delivered: 'August 8, 2020',
+    returnWindow: 'Sep 7, 2020',
+    orderNumber: '# 114-3941689-8772232',
+    writeProduct: true,
+    buyAgain: true,
+    viewItem: true,
+    handed: false,
+    support: false
```

Comment:
You should try to break this up into information about the order, and information about the individual items in the order, rather than grouping everything together in one object